### PR TITLE
feat(DB/Version): Add date on update

### PR DESCRIPTION
**Pull request by Kitzunu**
## Changes Proposed:
-  This is good and can open for opportunities where we can output current DB version with commands ingame etc.
- Introduce date column
-  Don't update and check column name
- Start to check latest date column instead
- insert date when importing DB

## How to Test the Changes:

1. run import.sh and db_assambler

## Known Issues and TODO List:

WHEN MERGED!!!!

We need to MANUALLY DELETE THIS FOR THE DB FILES:

```sql
SELECT date INTO @COLEXISTS
FROM version_db_db
ORDER BY date DESC LIMIT 1;
IF @COLEXISTS <> '"$oldVer"' THEN LEAVE proc; END IF;
```

when the CI has run the script for ONLY THESE FILES IN THE PR. Otherwise importing with db_assambler will break.

It works as it should after that and we don't need to touch it again.

## Tests Performed:

- Tested locally with import.sh and db_assambler.sh 

| # | C |
| :- | :- | 
| 1 | File that imported these changes (The ones you find in this PR) |
| 2 | A new file generated and imported after these changes |

WORLD:
image

CHARACTERS:
image

AUTH:
image

### CODE REVIEW

**Comment by FrancescoBorzi** [comment]
great idea @Kitzunu , even though I wonder if we could just read it from the column name instead

**Comment by Kitzunu** [comment]
I tried that, but didnt manage to pass it through the core

**Comment by Kitzunu** [comment]
TIP for merging:

After merge this, then in 1 PR you make the removals I described above + that you drop the column that has the name of a date. It is then easy to have the column name and needs less effort :D But that requires back to back merging

File: apps/db_pendings/import.sh [hunk 1]
```diff
@@ -65,14 +64,13 @@ function import() {
...
-                echo "WHERE TABLE_SCHEMA = DATABASE() AND TABLE_NAME = 'version_db_"$db"' AND COLUMN_NAME = '"$oldVer"';" >> "$newFile";
-                echo "IF @COLEXISTS = 0 THEN LEAVE proc; END IF;" >> "$newFile";
+                echo "SELECT date INTO @COLEXISTS" >> "$newFile";
+                echo "FROM version_db_"$db  >> "$newFile";
+                echo "ORDER BY date DESC LIMIT 1;"  >> "$newFile";
+                echo "IF @COLEXISTS <> '"$oldVer"' THEN LEAVE proc; END IF;" >> "$newFile";
             fi
 
             echo "$startTransaction" >> "$newFile";
-            echo "$updHeader" >> "$newFile";
```
Comment (Kitzunu) [inline apps/db_pendings/import.sh:75]: If you prefer I can revert this and only have the update part for now... for maybe faster merge :shrug:

**Review by locus313 — APPROVED** [review]

**Pull request merged**